src/main/java/conversion/BaseServlet.java

```diff
@@ -100,22 +104,41 @@ protected void doPost(final HttpServletRequest request, final HttpServletRespons
 
             individual.isAlive = true;
 
-            final Part filePart = request.getPart("file");
-            if (filePart == null) {
+            final byte[] fileBytes;
+            // Attempt to get the filename from parameters (null if not in params).
+            String fileName = request.getParameter("filename");
+            Part filePart = null;
+            
+            // Fail silently now and check for null later as the file might be passed via a url.
+            try {
+                filePart = request.getPart("file");
+            } catch (ServletException e) { }
```

Comment:
Not a fan of silent catches. Could we put a log message in there?
e.g. `LOG.fine("Request is not of type multipart/form-data (no file attached), checking if URL input provided.");`

It also provides more info as to why we're 'ignoring' the ServletException